.github/workflows/run-circle-mlir-build.yml

```
@@ -37,7 +37,7 @@ jobs:
     strategy:
       matrix:
         type: [ Debug, Release ]
-        ubuntu_code: [ focal, jammy ]
+        ubuntu_code: [ focal, jammy, noble]
```

It's just a minor formatting suggestion.. 😅

```suggestion
        ubuntu_code: [ focal, jammy, noble ]
```